administration/src/components/generation/PdfFactory.ts

```diff
@@ -73,7 +101,16 @@ export function generatePdf(models: CardActivateModel[], region: Region) {
     doc.setFont("NotoSans-Regular")
 
     for (let k = 0; k < models.length; k++) {
-        addLetter(doc, models[k], region)
+        let model = models[k];
+        let unsupportedChar = checkCardActivateModel(doc, model);
```

Comment:
```
        let unsupportedChar = checkCardActivateModel(doc, model);
```
The variable name is weird, as it is not clear from the function called what it means.
I think you should either rename the variable or the function.